plugins/org.eclipse.thym.core/src/org/eclipse/thym/core/platform/PlatformConstants.java

```diff
@@ -37,6 +39,12 @@
 	
 	public static final IPath[] CONFIG_PATHS = {new Path(PlatformConstants.DIR_WWW).append(PlatformConstants.FILE_XML_CONFIG),
 		new Path(PlatformConstants.FILE_XML_CONFIG) };
-
+	public static final IPath PLATFORMS_JSON_PATH = 
+			new Path(PlatformConstants.DIR_PLATFORMS).append(FILE_JSON_PLATFORMS);
+	
+	public static final String[] SUPPORTED_PLATFORMS = {
```

Comment:
supported platforms are pluggable. It is not a good idea to introduce a fixed list, instead we should use the extension point through `org.eclipse.thym.core.HybridCore.getPlatformSupports() `